app/src/main/res/values/strings.xml

```diff
@@ -287,30 +283,29 @@
 
     <!-- Notifications -->
     <string name="updated_matches_notification_title">
-        Woohoo! Beeline has new matches for you!
+        Psst! Beeline has new matches for you!
     </string>
     <string name="updated_matches_notification_text">
-        Once a day, Beeline runs an algorithm to pair the hotties of the world. The algorithm just
-        finished running. You\'ve got some hot peeps to check out!
+        Once a day, Beeline runs an algorithm to pair you beautiful people. The algorithm just
```

Comment:
I think both versions of the text are too long to actually show up in the system dialog for notifications. I'll try to get a screenshot to show you later.